src/schema/v2/alerts.ts

```diff
@@ -32,47 +89,142 @@ export const AlertType = new GraphQLObjectType<
   name: "Alert",
   fields: () => ({
     ...IDFields,
-    priceRange: {
+    acquireable: {
```

Comment:
Basically just added all of these - https://github.com/artsy/gravity/blob/6445b6ea90d2d7116b1082cb7baf2d3374238962/app/graphql/types/search_criteria_type.rb - which are all coming back in the JSON - https://github.com/artsy/gravity/blob/6445b6ea90d2d7116b1082cb7baf2d3374238962/app/models/search_criteria.rb#L435